src/lib/y2network/network_manager/connection_config_writers/wireless.rb

```diff
@@ -58,25 +47,84 @@ def write_eap_auth_settings(_conn)
         #
         # @see #write_eap_auth_settings
         # @see #write_psk_auth_settings
+        # @see #write_open_auth_settings
         # @see #write_shared_auth_settings
         def write_auth_settings(conn)
-          meth = "write_#{conn.auth_mode}_auth_settings".to_sym
+          auth_mode = conn.auth_mode || :open
+          meth = "write_#{auth_mode}_auth_settings".to_sym
           send(meth, conn) if respond_to?(meth, true)
         end
 
+        # Writes autentication settings for WPA-EAP networks
+        #
+        # @param conn [Y2Network::ConnectionConfig::Base] Configuration to write
+        def write_eap_auth_settings(conn)
+          # FIXME: incomplete
```

Comment:
Could you mention what is missing, please?

Reply:
Should be complete now, that is a leftover :)